MAINTAINERS.yml

```diff
@@ -3076,6 +3081,23 @@ Networking:
   tests:
     - net.wifi
 
+"Networking: HTTP":
+  status: maintained
+  maintainers:
+    - jukkar
+    - rlubos
```

Comment:
shouldn't @mrodgers-witekio be listed as a collaborator? :)

Reply:
Jukka suggested that I should create a PR to add myself after this is merged - I'm not too sure what the process should be but happy either way

Reply:
I felt it is more natural that Matt send a PR adding the collaborator status, this way I am also able to +1 the PR.